docs/fidesops/docs/guides/oauth.md

```diff
@@ -2,147 +2,92 @@
 
 In this section we'll cover:
 
-- An overview of the Fidesops Oauth flow
-- How to configure root client credentials
-- How to create additional oauth clients
-- Authorization scopes
-- How to create oauth tokens
-- Example auth flow
+- How to use the root client
+- Creating additional OAuth clients using the root client
+- Authorizing your client with scopes
+- Creating OAuth tokens
 
-Take me directly to [api docs](http://0.0.0.0:8080/docs#/OAuth) (you'll need to `make server` first)
+When you invoke a Fidesops API, you must pass an _access token_ as the value of the `Authorization` header. Furthermore, the token must included a _scope_ that gives you permission to do take an action on the API. For example, let's say you want to create a new Policy by calling `PUT /api/v1/policy`. The token that you pass to the `Authorization` header must include the `policy:create_or_update` scope.
 
-## Overview
+This document explains how to craft a properly-scoped access token.
 
-Authorization is the process of ensuring that only clients with proper permissions are able to perform specific actions.
+## Getting Started
 
-Fidesops supports Oauth authentication. Here's how that works:
+First, create an access token for the "root" client, included in the deployment by default. The root client's token is all-powerful: It contains all scopes so it can call any of the Fidesops APIs.
 
-1. A client pings Fidesops with its unique client id and secret
-2. If both the client id and secret are valid, Fidesops responds with an access token 
-3. The client provides that access token on each subsequent request for purposes of authorization
+To create the root token, you pass the `OAUTH_ROOT_CLIENT_ID` and `OAUTH_ROOT_CLIENT_SECRET` environment variables (which are automatically defined in your system) to the `POST /api/v1/oauth/token` endpoint. You also set the `grant_type` parameter to `client_credentials`:
```

Comment:
"which are automatically defined in your system" could read "which are specified in the Fidesops application config under the variables..."